.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: objective-c
 sudo: false
 os: osx
-osx_image: xcode9.2
+osx_image: xcode10
```

Comment:
Why 10 and not 10.3?

Reply:
We need to make sure WDA works with any minor version including 10.0